wpgraphql-ide.php

```diff
@@ -53,12 +53,25 @@ function user_lacks_capability(): bool {
 }
 
 /**
- * Checks if the current page is intended for the dedicated WPGraphQL IDE (non-drawer).
+ * Determines if the current admin page is a dedicated WPGraphQL IDE page.
+ *
+ * This function checks whether the current admin page is exclusively for either the new GraphQL IDE
+ * or the legacy GraphiQL IDE, distinguishing them from other admin pages where the IDE might be present
+ * in a drawer. It is designed to help in tailoring the admin interface or enqueuing specific scripts
+ * and styles on these dedicated IDE pages.
  *
  * @return bool True if the current page is a dedicated WPGraphQL IDE page, false otherwise.
  */
 function is_dedicated_ide_page(): bool {
+	return is_new_ide_page() || is_legacy_ide_page();
+}
 
+/**
+ * Checks if the current admin page is the new WPGraphQL IDE page.
+ *
+ * @return bool True if the current page is the new WPGraphQL IDE page, false otherwise.
+ */
+function is_new_ide_page(): bool {
```

Comment:
```suggestion
function is_ide_page(): bool {
```

It won't always be new, but it will always be the ide.

Let's maybe leave the new out?

Reply:
I went back and forth with this :) I agree lets remove it.